docs/docs/getting-started/deployment/configuration.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Configuration
+sidebar_label: Configuration
+---
+
+## Airy Core Configuration File
+
+The `infrastructure/airy.tpl.yaml` file contains examples of all possible configuration options. Copy and rename it to `infrastructure/airy.yaml` and modify it according to your environment:
+
+```bash
+cd infrastructure
+cp airy.tpl.yaml airy.yaml
+```
+
+We will guide you through the different sections so you can make the changes you are looking for. Keys can also be omited for services which you do not wish to configure.
+
+### Global
+
+- **appImageTag**
+
+  image tag of the container images for the `Airy Components`
```

Comment:
We should probably specific what kind of value they can expect here (as in the "official released" tags plus develop)